platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/DefaultConfigurationCacheDegradationController.kt

```diff
@@ -58,12 +75,26 @@ internal class DefaultConfigurationCacheDegradationController(
                         ?.sorted()
 
                     if (!taskDegradationReasons.isNullOrEmpty()) {
-                        collectedDegradationReasons[task] = taskDegradationReasons
+                        collectedDegradationReasons[DegradationContext.Task(task)] = taskDegradationReasons
                     }
                 }
             }
         }
     }
 
+    private fun collectFeatureDegradationRequests() {
```

Comment:
💅 There is some ambiguity in between `requests`/`reasons` in context of this class. Both may be collected for sure. But given that enclosing `collectDegradationReasons` method is about `reasons`, I believe this one should be similarly name `collectFeatureDegradationReasons`